src/client/microsoftAuth.js

```diff
@@ -52,21 +72,132 @@ module.exports = async (client, options) => {
   }
 
   const session = {
-    accessToken: MineServicesResponse.access_token,
+    accessToken: mcAccessToken,
     selectedProfile: profile,
     availableProfile: [profile]
   }
   client.session = session
   client.username = MinecraftProfile.name
-  options.accessToken = MineServicesResponse.access_token
+  options.accessToken = authConstants.MineServicesResponse.access_token
```

Comment:
This line is incorrect and will cause the client to error out. 

It should be `options.accessToken = mcAccessToken` because MineServesResponse is a undefined property in authConstants.
(Found during testing)